### DISCUSSION TITLE
The last analog counter (xo.oo1) is digital, no pointer

### DISCUSSION THREAD

**Discussion by d-coimbra** [jomjol/AI-on-the-edge-device#2690] [Q&A]
**Is there a way to include this digit on measure?**

![alg_roi](https://github.com/jomjol/AI-on-the-edge-device/assets/78891216/315c7b08-3323-47b8-b06c-0e2fc4f84ed7)

**I can't determine lower value than x1**

![config_roi](https://github.com/jomjol/AI-on-the-edge-device/assets/78891216/49972041-1b6e-4af6-a185-2b91e7a8816a)

**Comment by friedpa** [comment]
I cannot even read the numbers with my eyes, so I would not expect that the reading can be done. But maybe you can increase the contrast and get rid of the reflections in the lowere part. Try to use a diffuser (e.g. milky adhesive tape)

**Reply by caco3** [reply]
The firmware does not support this directly.
You could use the 0.001 digit as a 2nd meter and add it up externally. but you will miss the whole consistency checks.
My suggestion would be to simply keep it away. The 0.01 pointer should give you an accuracy of almost 0.001, especially with https://jomjol.github.io/AI-on-the-edge-device-docs/Parameters/#PostProcessing-NUMBER.ExtendedResolution

**Reply by d-coimbra** [reply]
Thanks Caco3...
If I had an option, I would do it
I'll ignore it.
